### DISCUSSION TITLE
How to split "forensic" into syllables?

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#24] [Q&A] [answered]
For me "fo-ren-sic" would be a natural way - is it wrong? I guess version can be disregarded as I'm not a native English speaker, also I've looked up - this word stems from latin "forum" and then it would make sense to  assign the first syllable to "for".

The dictionaries in plover split as for-en-sic + 1 abbreviated entry.

On the other hand. -ic is a suffix! So if I follow the rules given in CH15, I would first split out the -ic as suffix and be left with "forens", which I would split as fo-rens, yielding the outline "TPO/REGPBS/KWREUBG" (does not work).

However - TPOR/EPBS/KWREUBG is working <- but how come the second syllable does not contain KWR? - shouldn't all syllables starting with a vowel begin with KWR?

Should the outline for the CH15 exercises follow the logic of the lesson and split it like for/ens/ic instead of for/en/sic?

**Comment by aerickt** [comment, marked as answer]
In my accent of English, I tend to emphasize the r, and that has influenced the Lapwing dictionary. This is why `TPO/REPB/SEUBG` is missing from the dictionary. You can either train yourself to keep the r in the previous stroke (i.e. `TPOR/KWREPB/SEUBG` rather than `TPO/REPB/SEUBG`), or add the missing entries to your own dictionary. It is very rare that a word in the dictionary can only be split before the r. Also, this may be a little advanced, but I would suggest using the abbreviated `TPREPB/SEUBG` stroke since it saves a stroke, drops the unstressed o, and keeps the `REPB` sound together.

What you should not do, however, is use a suffix stroke in this situation: https://lapwing.aerick.ca/Chapter-13.html#kwr-suffixes

> Use these suffixes only to modify a root word. Ensure that the root word relates to the full word semantically.

I would not use the `KWREUBG` suffix since I don't consider "forens" a word.

**Reply by evgenyfadeev** [reply]
Cool, thanks I'll keep in mind not splitting before the r! 

Why in the dictionary outline TPOR/EPBS/KWREUBG, the EPBS is not starting with KWR? Isn't -ens- starting with a vowel?

**Reply by aerickt** [reply]
That's really just an accident due to all those strokes containing an affix.
- `TPOR` is the "for^" prefix
- `EPB` is the "en^" prefix, therefore `EPBS` is "ens"
- `KWREUBG` is the "^ic" suffix

If you put them all together, you get "for" + "ens" + "ic".

**Reply by evgenyfadeev** [reply]
I see how it all comes together, but shouldn't syllables after the first that start with a vowel start with KWR?

Actually - for is hardly a prefix in this case - the word stems from FORUM - a Latin word. -ensic is prabably a changed latin suffix. but the root of the word at least contains for-, that is not a suffix. there is no Postensic, proensic, etc.

How can a word contain only prefixes and suffixes and no root? Perhaps the linguistic logic does not apply here...

If en is a prefix and it has a fixed outline then you don't add the KWR indeed.

**Reply by aerickt** [reply]
> I see how it all comes together, but shouldn't syllables after the first that start with a vowel start with KWR?

Yes, you are right. That's why `TPOR/KWREPB/SEUBG` is considered correct and `TPOR/EPBS/KWREUBG` isn't. However, any outline that contains `KWREUBG` would automatically be considered incorrect since this is an improper use of a suffix stroke.

> Actually - for is hardly a prefix in this case - the word stems from FORUM - a Latin word. -ensic is prabably a changed latin suffix. but the root of the word at least contains for-, that is not a suffix. there is no Postensic, proensic, etc.

You are right here as well. However, the outline `TPOR/KWREPB/SEUBG` is constructed by using syllabic splitting. This method does not care if any of the strokes it forms are affixes. It is only a coincidence* that `TPOR` is also defined as a prefix stroke.

*It's not really a coincidence, but a decision to separate the word "for" (briefed as `TP-R`) and the cluster "for^" that is used in words such as "forgive". If there was no distinction between the word and the cluster (i.e. "for" defined as `TPOR` and "forgive" defined as `TPOR/TKPWEUF`), you could run into issues where you can only write "forgive" and not "for give" (e.g. "thank you forgiving me this gift").
